tests/01-handshake/testbench.cc

```diff
@@ -0,0 +1,91 @@
+
+#include <iomanip>
+#include <iostream>
+
+#include <SwAxi.hh>
+
+class Ram : public sw_axi::Slave {
+public:
+    virtual int handleWrite(const sw_axi::Buffer *buffer) {
+        return 0;
+    }
+    virtual int handleRead(sw_axi::Buffer *buffer) {
+        return 0;
+    }
+};
+
+int main(int argc, char **argv) {
+    using namespace sw_axi;
+
+    Bridge bridge;
+
+    if (bridge.connect() != 0) {
+        std::cerr << "Unable to connect to the simulator" << std::endl;
+        return 1;
+    }
+
+    std::cerr << "Connected to the simulation system" << std::endl;
+    const SystemInfo *info = bridge.getRemoteSystemInfo();
+    std::cerr << "Name:     " << info->name << std::endl;
+    std::cerr << "Pid:      " << info->pid << std::endl;
+    std::cerr << "URI:      " << info->uri << std::endl;
+    std::cerr << "Hostname: " << info->hostname << std::endl;
+
+    Ram *ram = new Ram;
+    IpConfig ramConfig = {
+            .name = "Soft-RAM",
+            .startInterrupt = 0,
+            .endInterrupt = 0,
+            .startAddress = 0x1000,
+            .endAddress = 0x2000,
```

Comment:
Probably want to end at 0x1FFF to match hardware convention.

Reply:
Okay, so the conversion in hardware is generally for the end limit of an address range to be included? I don't mind either way, just want to be consistent on both sides. From what I have seen, in software people tend to exclude the end limit - results with "rounder" numbers, which is easier to interpret.

Reply:
OK, I'd opt to change this to `(addr, size)` pair instead if that more consistent with the hardware convention.